[WIP] Adding German language notebooks (#65)
nvidia-riva/tutorials: New-language-adaptation/German/deployment/README.md

   --offline \
   --name=citrinet-1024-de-DE-asr-offline \
   --ms_per_timestep=80 \
   --featurizer.use_utterance_norm_params=False \
   --featurizer.precalc_norm_time_steps=0 \
   --featurizer.precalc_norm_params=False \
   --chunk_size=900 \
   --left_padding_size=0. \
   --right_padding_size=0. \
   --decoder_type=flashlight \
   --decoding_language_model_binary=/servicemaker-dev/speechtotext_de_de_lm_vdeployable_v2.0/riva_de_asr_set_2.0_4gram.binary \
   --decoding_vocab=/servicemaker-dev/speechtotext_de_de_lm_vdeployable_v2.0/dict_vocab.txt \
   --flashlight_decoder.lm_weight=0.2 \
   --flashlight_decoder.word_insertion_score=0.2 \
   --flashlight_decoder.beam_threshold=20. \
   --wfst_tokenizer_model=/servicemaker-dev/inverse_normalization_de_de_vdeployable_v1.0/tokenize_and_classify.far \
   --wfst_verbalizer_model=/servicemaker-dev/inverse_normalization_de_de_vdeployable_v1.0/verbalize.far \
   --language_code=de-DE

riva-deploy -f /servicemaker-dev/citrinet-1024-de-DE-asr-offline.rmir /data/models
```

The `riva-build` command takes in an acoustic model in `.riva` format, the inverse text normalization models in `.far` format, and a n-gram binary language model file.

> Note: See Riva documentation for build commands for streaming ASR services.

Reviewer: ```suggestion
Note: Refer to the [Riva ASR Pipeline Configuration documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html#asr-pipeline-configuration) for build commands for streaming ASR services.
```

> #### Build and deploy and punctuation and capitalization model

Reviewer: ```suggestion
#### Build and Deploy and Punctuation and Capitalization Model
```

> When doing ASR, the Riva server will look for a punctuator model that matches the language in the ASR request config.

Reviewer: ```suggestion
When doing ASR, the Riva server looks for a punctuator model that matches the language in the ASR request config.
```

The punctuator model can be built and deployed with:

```bash
riva-build punctuation -f \
   /servicemaker-dev/de_punctuation_1_0.rmir  \
   /servicemaker-dev/punctuationcapitalization_de_de_bert_base_vdeployable_v1.0/de_punctuation_1_0.riva \
   --language_code=de-DE

riva-deploy -f /servicemaker-dev/de_punctuation_1_0.rmir /data/models 
```

> ## Start Riva server

Reviewer: ```suggestion
## Start the Riva Server
```

Reviewer: we should add the command to start the server

> That concludes the building and deployment of the Riva German ASR service. Now you can start the Riva server.

Reviewer: ```suggestion
That concludes the building and deployment of the Riva German ASR service. 
```

Reviewer: This sentence should come before line 149 - Start the Riva Server.
